ASM/c/dungeon_info.c

```diff
@@ -208,11 +211,12 @@ void draw_dungeon_info(z64_disp_buf_t *db) {
         }
         text_flush_size(db, font_width, font_height, 0, 0);
 
-        left += (8 * font_width) + padding;
+        left += (11 * font_width) + padding;
 
         // Draw keys
 
         if (show_keys) {
+
```

Comment:
Please don't introduce arbitrary whitespace changes.

Reply:
Removed whitespace, however all existing ones had the same white space.   I did not adjust those